airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -40,7 +41,10 @@ class GoogleDataprepHook(BaseHook):
     def __init__(self, dataprep_conn_id: str = "dataprep_conn_id") -> None:
         super().__init__()
         self.dataprep_conn_id = dataprep_conn_id
-        self._url = "https://api.clouddataprep.com/v4/jobGroups"
+        conn = self.get_connection(self.dataprep_conn_id)
+        extra_dejson = conn.extra_dejson
+        self._token = extra_dejson.get("token")
```

Comment:
```suggestion
        self._token = extra_dejson.get("extra__dataprep__token")
```
According to documentation and Airflow best practices

Reply:
Yes. Thank you.